src/transaction/components/TransactionMessage.tsx

```diff
@@ -0,0 +1,74 @@
+import { cn, color, text } from '../../styles/theme';
+import { useTransactionContext } from './TransactionProvider';
+import type { TransactionMessageReact } from '../types';
+
+export function TransactionMessage({ className }: TransactionMessageReact) {
+  const { address, contracts, errorMessage, gasFee, isLoading, transactionId } =
+    useTransactionContext();
+
+  const containerClassName = cn(
+    'flex gap-2 pt-2 justify-between w-full',
+    text.label2,
+    className,
+  );
+  const testID = 'ockTransactionMessage_Message';
+
+  if (transactionId) {
+    return (
+      <div className={containerClassName} data-testid={testID}>
+        <p>Successful!</p>
```

Comment:
So it's interesting how we want to build this because, we want to have two groups

```tsx
<Transaction
  address={accountAddress}
  ​​className="custom-style"
  contracts={contractsToCall}
  onError={handleTransactionError}
  onStatusChange={handleTransactionStatusChange}
  sponsorOptions={sponsorOptions}
>
  <TransactionButton text={optionalButtonText} />
  <TransactionGasFee>
    <TransactionGasFeeLabel />
    <TransactionGasFeeEstimated />
    <TransactionGasFeeSponsoredBy />
  </TransactionGasFee>
  <TransactionStatus>
    <TransactionStatusLabel />
    <TransactionStatusAction />
  </TransactionStatus>
</Transaction>
```

We want
```tsx
<TransactionGasFee>
    <TransactionGasFeeLabel />
    <TransactionGasFeeEstimated />
    <TransactionGasFeeSponsoredBy />
  </TransactionGasFee>
```
to be used for the initial flow, and than 
```
 <TransactionStatus>
    <TransactionStatusLabel />
    <TransactionStatusAction />
  </TransactionStatus>
```
for the second part of the flow.

And if you see the sub components will get the information and we can customize their style.